docs/src/introduction/components.md

```diff
@@ -0,0 +1,55 @@
+# Components
```

Comment:
Remove this subsection and put the corresponding constructors in OmniPaxos and BLE instead.